lib/rules/aria-dpub-role-fallback-matches.js

```diff
@@ -0,0 +1,10 @@
+var role = node.getAttribute('role');
+return [
+		'doc-backlink',
+		'doc-biblioentry',
+		'doc-biblioref',
+		'doc-cover',
+		'doc-endnote',
+		'doc-glossref',
+		'doc-noteref'
+	].some(r => r === role);
```

Comment:
You could do a `.includes` instead. Saved you the callback.

Reply:
doh, right :-)